assertsprocessor/service_queue.go

```diff
@@ -0,0 +1,64 @@
+package assertsprocessor
+
+import (
+	"math"
+	"sync"
+)
+
+type serviceQueues struct {
+	config        *Config
```

Comment:
I was in two minds about it but decided to keep the pointer. It looks like both pointer and `int` are 8 bytes on 64bit OS. So it's more efficient to use the pointer instead of two `int` types.